index.html

```diff
@@ -15,5 +17,78 @@
 
 <body>
 <!-- Use your own code or past solution for Great Idea Web Page HTML here! -->
+<nav>
+ <a href="/services.html">Services</a>
+ <a href="#">Product</a>
+ <a href="#">Vision</a>
+ <a href="#">Features</a>
+ <a href="#">About</a>
+ <a href="#">Contact</a>
+<img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+ </nav>
+
+<div class="on-demand-get-started-pic">
+<div class="demand-start">
+
+ <h1 class="on-demand">
+ Innovation<br>
+ On<br>
+ Demand
+ </h1>
+ <section class="get-started">Get Started</section>
+ </div>
+<img class="circlecode" src="img/header-img.png" alt="Image of a code snippet.">
+</div>
+
+<div class="box2">
+<div class="feature-about-box">
+
+
+<div class="features">
+
+<h2 >Features</h2>
+<p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+</div>
+
+
+<div class="about">
+<h2 >About</h2>
+<p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+</div>
+</div>
+<img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+
+<div class= services-product-vision-box> 
+<div class="services" >
+<h2 >Services</h2>
+<p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+</div>
+<div class="product" >
+
+<h2 >Product</h2>
+<p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+</div >
+<div class="vision">
+<h2 >Vision</h2>
+<p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+</div>
+</div>
+
+
+
+</div>
+
+
+<div class="contact">
+<h2>Contact</h2>
+ <address>
+ 123 Way 456 Street <br>
+ Somewhere, USA<br>
+ 1 (888) 888-8888 <br>
+ sales@greatidea.io<br>
+ </address>
+</div>
+<footer class="foot">Copyright Great Idea! 2018 </footer>
```

Comment:
I would use the footer as the main container and have a div inside with the copyright info